app/templates/_test-ava.js

```diff
@@ -2,7 +2,7 @@ import test from 'ava';
 import <%= camelModuleName %> from './index';
 
 test('should <%= camelModuleName %>', (t) =>
-  t.same(<%= camelModuleName %>('unicorns'), 'unicorns'));
+  t.is(<%= camelModuleName %>('unicorns'), 'unicorns'));
 
 test('should <%= camelModuleName %> invalid input', (t) =>
-  t.same(<%= camelModuleName %>(), undefined));
+  t.is(<%= camelModuleName %>(), undefined));
```

Comment:
`t.same` is a deep equalization. Why use it for primitive values?

You don't use `.deepEqual` method in `mocha` and `tape` templates.